### DISCUSSION TITLE
Register as default calculator in Windows

### DISCUSSION THREAD

**Discussion by cousteaulecommandant** [Qalculate/qalculate-gtk#418] [Ideas]
In Windows, pressing the "Calculator" button on my keyboard opens the MS Calculator.
Browsing the options, I found that in "Default apps > Choose default apps by protocol" I can assign an app to the CALCULATOR protocol, which I suppose is what opens when you press the Calculator button (or type `calculator://` in a browser).  Currently, I can choose between MS Calculator and... well, that's the only app I can choose.  There's no way to add custom executables; they need to be apps that somehow state that they support that protocol.

Do you think Qalculate could register itself as an app that supports the `calculator:` protocol when installed, or otherwise give an option to be set as the default calculator?  (I have no idea what's the exact procedure to do this; just leaving it as a suggestion)

**Comment by foreachthing** [comment]
Here's help: https://winaero.com/change-what-extra-app-keys-do-for-the-keyboard-in-windows-10-8-and-7/amp/
